x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/security/authz/14_cat_indices.yml

```diff
@@ -164,7 +164,7 @@ teardown:
 
   - match:
       $body: |
-        /^(yellow  \s+
+        /^((yellow|green)  \s+
```

Comment:
Due to different index health rules, this index will be green on serverless even though it's yellow on stateful.
The test shouldn't care - it's just checking that correct set of indices are returned in the cat API.